core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/print/PrintProperty.java

```diff
@@ -19,33 +19,65 @@
 
 package de.valtech.aecu.core.groovy.console.bindings.actions.print;
 
-import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import java.util.Arrays;
+
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.Property;
+import javax.jcr.RepositoryException;
+import javax.jcr.Value;
 
 import org.apache.sling.api.resource.Resource;
-import org.apache.sling.api.resource.ValueMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import javax.annotation.Nonnull;
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
 
 /**
  * Action for printing the name and value of a given property.
  *
+ * @author sravan
  * @author Roxana Muresan
  */
 public class PrintProperty implements Action {
 
-    private String propertyName;
+	private static final Logger LOG = LoggerFactory.getLogger(PrintProperty.class);
+
+	private final String NOT_DEFINED = " not defined";
```

Comment:
Since these constants are used only once you can avoid them and directly use the text in the code for better readability.